### PR TITLE
Document changing the window's resizable status at runtime

### LINKED ISSUE #84876: ProjectSetting set "display/window/size/resizable" does not take effect
### Godot version

Godot v4.2.beta.mono (fee6df76a)

### System information

Godot v4.2.beta.mono (fee6df76a) - Windows 10.0.22631 - Vulkan (Forward+) - dedicated NVIDIA GeForce RTX 3060 Laptop GPU (NVIDIA; 31.0.15.3713) - 12th Gen Intel(R) Core(TM) i7-12700H (20 Threads)

### Issue description

ProjectSetting set "display/window/size/resizable" does not take effect

### Steps to reproduce

1.set display/window/size/resizable=true in editor 
2.call ProjectSettings.SetSetting("display/window/size/resizable",false) in script
3.Window can still be resized

### Minimal reproduction project

"N/A"

### PR DESCRIPTION
**Pull request by Calinou**
- This closes https://github.com/godotengine/godot/issues/84876.

### CODE REVIEW

**Review by AThousandShips — APPROVED** [review]

File: doc/classes/ProjectSettings.xml [hunk 1]
```diff
@@ -810,7 +810,9 @@
 			Main window can't be focused. No-focus window will ignore all input, except mouse clicks.
 		</member>
 		<member name="display/window/size/resizable" type="bool" setter="" getter="" default="true">
-			Allows the window to be resizable by default.
+			If [code]true[/code], allows the window to be resizable by default.
+			[b]Note:[/b] This property is only read when the project starts. To change whether the window is resizable at runtime, set [member Window.unresizable] instead on the root Window, which can be retrieved using [code]get_viewport().get_window()[/code]. [member Window.unresizable] takes the opposite value of this setting.
```
Comment (bruvzg) [inline doc/classes/ProjectSettings.xml:814]: This is true for pretty much any project setting (at least window releated).

Comment (Jordyfel) [reply doc/classes/ProjectSettings.xml:814]: `get_window()` is available in base `Node`, like `get_viewport()`

Comment (akien-mga) [reply doc/classes/ProjectSettings.xml:814]: So what should we do here? It seems a bit weird indeed to flag it specifically here when it applies to all other similar settings.

Comment (Calinou) [reply doc/classes/ProjectSettings.xml:814]: > So what should we do here? It seems a bit weird indeed to flag it specifically here when it applies to all other similar settings.

The policy until now has been to add this note to every project setting where relevant (as some *do* have an affect when change at runtime). I agree it gets repetitive, but until we get a macro system for documentation, I'm not sure how we can avoid this while ensuring the documentation is as informative as possible.

**Comment by akien-mga** [comment]
Thanks!

File: doc/classes/ProjectSettings.xml (hunk 1 above)
Comment (akien-mga) [reply doc/classes/ProjectSettings.xml:814]: Hm right, then doing it manually might still be the best option.

**Comment by YuriSizov** [comment]
Cherry-picked for 4.2.2.

**Comment by YuriSizov** [comment]
Cherry-picked for 4.1.4.

**Pull request merged**